packages/devtools-reps/src/reps/tests/date-time.js

```diff
@@ -41,6 +43,6 @@ describe("test invalid DateTime", () => {
       })
     );
 
-    expect(renderedComponent.text()).toEqual("Invalid Date");
```

Comment:
I'm not sure why this change is needed?

Reply:
so, turns out putting an invalid date in the constructor does not throw, it just produces an invalid date. it was the `.toISOString()` call that was then throwing.
So maybe what we could do is keep the call to `toISOString()` just so it can throws and detect Invalid date.

```js
  try {
    const dateObject = new Date(grip.preview.timestamp);
    // this call throws if we are dealing with an invalid date.
    dateObject.toISOString();
    date = span(
      {
        "data-link-actor-id": grip.actor,
        className: "objectBox"
      },
      getTitle(grip),
      span({ className: "Date" }, dateObject.toString())
    );
  } catch (e) {
    date = span({ className: "objectBox" }, "Invalid Date");
  }
```